.travis.yml

```diff
@@ -2,6 +2,7 @@ language: php
 
 php:
   - 5.4
+  - 5.5
   - 5.6
   - 7.0
```

Comment:
We would. But is that necessary? I mean we already run Symfony 3.0 tests with other PHP versions.